Gem/Code/Source/Components/UI/UiSettingsComponent.cpp

```diff
@@ -258,40 +258,83 @@ namespace MultiplayerSample
         // Get the current fullscreen state. Unlike the other settings, we'll get this from the current window state so that we
         // handle things like Alt-enter that can change our windowing state regardless of what our user settings thinks.
 
-        // Start by defaulting to the user setting.
+        // Start by getting the user setting as our default state.
```

Comment:
nit: avoid personal pronouns in comment, its not clear to readers who "ours" is, o3de? the sample?